### ISSUE TITLE
Test issues

### ISSUE THREAD

**Issue by joshuabamboo** [learn-co-curriculum/javascript-fix-the-scope-lab#4]
[`file`](https://github.com/learn-co-curriculum/javascript-fix-the-scope-lab/blob/master/test/index-test.js#L38) in the test is not defined. The tests were recently updated. [ Looks like `file` was taken out here](https://github.com/learn-co-curriculum/javascript-fix-the-scope-lab/commit/bd71d0e80f24be5b41f148884ca8906c4df41253#diff-1c5726b71a63db52a608b0c2c9726644L12) 

Another testing note. [This test](https://github.com/learn-co-curriculum/javascript-fix-the-scope-lab/blob/master/test/index-test.js#L15) returns this error even when it is solved.

```  
1) yourAnimal() does not hard-code the answer:
     TypeError: Cannot read property 'to' of undefined
      at Context.<anonymous> (test/index-test.js:15:38)

Same for this one:

  2) funkyFunction() returns a function:
     TypeError: Cannot read property 'be' of undefined
      at Context.<anonymous> (test/index-test.js:28:31)
```

@PeterBell

**Comment by pletcher** [comment]
Thanks for opening this issue!

Keep in mind that this space is for specific issues related to the content of this lesson.

If you're reporting on the content of this lesson, it'd be super helpful if you could include a stack trace of any errors that you're seeing.

If you'd like to report a bug about the Learn IDE, the Ask a Question form, opening a file, or the lights not working on Learn, please report a bug directly on https://learn.co. (You can find the bug report form by clicking on your avatar in the upper right, and then selecting "Report A Bug".)

Please be sure to include steps to reproduce (be as detailed as possible) and any relevant information about your environment. (Are you using the Learn IDE? Coding on a refurbished toaster?) We'll be sure to figure out what's going on — and to jump in and help — just as soon as we can. Thanks!

**Comment by pletcher** [comment]
Fixed via #5
